scripts/SREP/get-targets-down/metadata.yaml

```diff
@@ -0,0 +1,40 @@
+file: script.sh
+name: example # name should be the same as the subdir, eg: SRE/example in this case
+description: Example script
+author: qhua948
+allowedGroups: 
+  - SREP
+  - CSSRE
+  - CEE
+  - MTSRE
+rbac:
+    roles:
+      - namespace: "openshift-monitoring"
+        rules:
+          - verbs:
+            - "get"
+            apiGroups:
+            - "route.openshift.io"
+            resources:
+            - "routes"
+          - verbs:
+            - "get"
+            apiGroups:
+            - ""
+            resources:
+            - "serviceaccounts"
+          - verbs:
+            - "get"
```

Comment:
Could you try remove those lines, we by default have cluster-reader perms. https://github.com/openshift/managed-cluster-config/tree/master/deploy/osd-backplane-managed-scripts

Reply:
(I would guess Manuel didn't know as he did some automation to generate the permissions)

Also, having this cluster reader role is a bit confusing

Reply:
The reason for this is all teams already have cluster reader role by default, so folks can only care about the "edit" perms, but secret is an exception, cause we don't allow folks to access secret by default.

Reply:
Guessing which permissions may be there by default makes testing even harder, but good to know :). Would it hurt to keep them here for documentation purposes?

Reply:
I guess this makes testing on the developers side much cleaner, and less dependant on cluster permissions

Reply:
This does make sense to me.